### ISSUE TITLE
Plugin update during download doesn't restart pyload

### ISSUE THREAD

**Issue by manacker** [pyload/pyload#2118]
Today I got a plugin update during a download:

```
26.10.2015 13:34:10 INFO    ADDON UpdateManager: pyLoad is up to date!
26.10.2015 13:34:11 INFO    ADDON UpdateManager: New version of plugin: CRYPTER YoutubeComFolder (v1.05 -> v1.06)
26.10.2015 13:34:12 INFO    ADDON UpdateManager: New version of plugin: HOOK XFileSharing (v0.50 -> v0.51)
26.10.2015 13:34:12 INFO    ADDON UpdateManager: New plugin: HOSTER RyushareCom (v0.16)
26.10.2015 13:34:12 INFO    ADDON UpdateManager: *** Plugins updated ***
26.10.2015 13:34:12 WARNING ADDON UpdateManager: You have to restart pyLoad to use the updated plugins
26.10.2015 13:34:12 WARNING ADDON UpdateManager: pyLoad restart scheduled | Downloads are active, pyLoad restart postponed once the download is done
```

This stopped new downloads, but didn't trigger a reload after the current downloads were done. So the download queue got stuck.

**Comment by snilt** [comment]
all_downloads_processed() is not called when the server is paused
and there are still queued files.

This seems to work for me, UpdateManager.py:
```
def periodical_task(self):
    if self.do_restart is True:
        if self.pyload.threadManager.pause and not self.pyload.api.statusDownloads():
            self.pyload.api.restart()
            return
    ........
    ........
    ........
```

**Comment by GammaC0de** [comment]
Thanks 👍
